Modules/CaPTkSurvival/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+include_directories(src/internal)
+
+mitk_create_module(
+  INCLUDE_DIRS PUBLIC include #third_party/yaml-cpp third_party/jsoncpp/include
+  PACKAGE_DEPENDS ITK VTK Qt5|Core+Widgets OpenCV
+  DEPENDS PUBLIC MitkCaPTkCommon #MitkCore
+)
+
+# standard cmake way of copying files -- doesn't copy into the packaged install
+#file(COPY resources/models/survival_model DESTINATION ${CMAKE_BINARY_DIR}/bin/models)
```

Comment:
Why is that commented out?

Reply:
I thought this wouldn't be required with MITK_INSTALL but I was wrong -- fixed.
Done, will update this PR momentarily